kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/FileConversionServiceImpl.java

```diff
@@ -132,12 +140,21 @@ public void startIfEnabled() {
 
                         workers.submit(() -> {
 
-                            log.debug("Setting item with ref {} to IN_PROGRESS ...", ref);
-                            item.setStatus(FileConversionQueueItem.Status.IN_PROGRESS);
-                            item.setAttempts(item.getAttempts() + 1);
-                            item.setLastAttemptStarted(Instant.now());
+                            // This should update the item to IN_PROGRESS and flush the change so
+                            // that other workers can see it. This is the mutex, in effect.
+                            (new TransactionTemplate(transactionManager)).execute(new TransactionCallbackWithoutResult() {
```

Comment:
You don't need to to create a template all the time can simply use the existing.